pages/d/[id].tsx

```diff
@@ -2,8 +2,8 @@ import { GetServerSideProps } from "next";
 
 import App from "../../src/app";
 
-import { spaceExists } from "../../replicache-nextjs/api";
-import { useReplicache } from "../../replicache-nextjs/frontend/use-replicache";
+import { spaceExists } from "replicache-nextjs/out/backend";
```

Comment:
Instead of having the export be `out/backend` can we have it be just `backend`?

Reply:
This is a TS issue that ignored exports in package.json, I believe is fixed in 4.7. I'll try it out.

Reply:
I tried my best to fix this issue, but was not successful there is a 200comment thread about this here 
https://github.com/microsoft/TypeScript/issues/33079. @grgbkr recommended I just rename the out to lib instead and we can figure it out later.